playbooks/openshift-logging/private/redeploy-certificates.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Logging Install Checkpoint Start
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: Set Logging install 'In Progress'
+    run_once: true
+    set_stats:
+      data:
+        installer_phase_logging:
+          title: "Logging Install"
+          playbook: "playbooks/openshift-logging/config.yml"
+          status: "In Progress"
+          start: "{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}"
+
+- name: OpenShift Aggregated Logging
+  hosts: oo_first_master
+  tasks:
+  # make sure ES is green first?
+
+  - name: Removing prior certificates
+    file:
+      state: absent
+      path: "{{openshift.common.config_base}}/logging/*.{{file_extension}}"
+    with_items:
+    - crt
+    - key
+    - jks
+    - csr
+    loop_control:
+      loop_var: file_extension
+
+  - name: Regenerating certificates
+    include_role:
+      name: openshift_logging
+      tasks_from: generate_certs.yaml
+      vars:
+        generated_certs_dir: "{{openshift.common.config_base}}/logging"
+
+  - name: Regenerating secrets
+    include_role:
+      name: "{{logging_role}}"
+      tasks_from: generate_secret.yaml
+      vars:
+        generated_certs_dir: "{{openshift.common.config_base}}/logging"
+    with_items:
+    - openshift_logging_elasticsearch
+    - openshift_logging_kibana
+    - openshift_logging_curator
+    - openshift_logging_fluentd
+    - openshift_logging_mux
+    loop_control:
+      loop_var: logging_role
+
+  # restart all components except curator since it is a cronjob
+  # fluentd and mux will automatically reload the remounted secret
```

Comment:
?  I think you still need to restart fluentd and mux - note that mux is a dc, not a daemonset

Reply:
fluentd we've seen will reload the certs. but i mux will need a restart

Reply:
ah, I see - if it is configured to reload connections, it will pick up the new certs when it reloads